README_DEVELOPERS.md

```diff
@@ -1,9 +1,20 @@
 # Testing
 
+## Build and run docker image with browser
```

Comment:
I suggest splitting this into 2 paragraphs:
`## Build docker image (once)`

'## Run docker'